.changeset/wide-roses-hang.md

```diff
@@ -0,0 +1,5 @@
+---
+'rock': patch
+---
+
+move fingerprint debugging logs behind verbose flag
```

Comment:
Not sure if this should be patch or minor since it is technically breaking change since it changes output, but very minor

Reply:
patch, changesets don't take 0.x into account :<